packages/lit-dev-tools-cjs/src/api-docs/transformer.ts

```diff
@@ -412,6 +412,64 @@ export class ApiDocsTransformer {
     }
   }
 
+  /**
+   * Remove fields that we don't need for rendering. This makes reading diffs
+   * much easier, since we check the generated JSON file in.
+   */
+  private prunePageData(node: unknown) {
+    if (node instanceof Array) {
+      for (const item of node) {
+        this.prunePageData(item);
+      }
+    } else if (typeof node === 'object' && node !== null) {
+      // Method comments are duplicated both at the root of the node, and also
+      // inside its signature. Remove the one from the signature.
+      if (
+        (node as ExtendedDeclarationReflection).comment &&
+        (node as ExtendedDeclarationReflection).signatures?.[0]?.comment
+      ) {
+        delete (node as ExtendedDeclarationReflection).signatures?.[0]?.comment;
+      }
+      for (const [key, val] of Object.entries(node)) {
+        // Prune the child first, so that our "empty arrays and objects" check
+        // works more aggressively.
+        this.prunePageData(val);
+        if (
```

Comment:
Non actionable because it could be a larger change than I understand. But instead of justifying why we are opting things out, this could also be modeled as an allow-list, with justification for why something is left in. Not sure of the pro/cons but it would ensure that if typedoc added something, it wouldn't automatically be pulled in.

Reply:
The thing is that we don't build the data structure to begin with, we get it from TypeDoc and then augment it. So to do an allowlist approach, we'd need to re-build the data structure which I think would be more complicated.